pages/pipelines/permissions.md

```diff
@@ -135,6 +135,8 @@ From the Member Permissions page, organization admins can toggle whether or not
 
 If your organization has teams enabled, the pipeline creation permissions are managed at a team level. Pipeline creation permission controls can be found on the Teams Settings page. Without teams enabled, the pipeline creation permission control can be found on the Member Permissions page.
 
+Note:  The Team-Level permissions on a given pipeline, will override the lower priviledge member permisions in the team settings.
```

Comment:
I find this quite hard to parse. Could you try expanding it a bit more or perhaps providing an example? Thinking like:

> Note that team-level permissions override member permissions. For example, if the same user has a team level permission to X on a pipeline but only a X member permission on that pipeline, they can...